### ISSUE TITLE
`TypeError: Cannot read properties of null (reading '1')` when running with --cache

### ISSUE THREAD

**Issue by P4** [raineorshine/npm-check-updates#1276]
- [x] I have searched for [similar issues](https://github.com/raineorshine/npm-check-updates/issues)
- [x] I am using the latest version of `npm-check-updates` (`ncu --version` outputs `16.7.10`)
- [x] I am using `node >= 14.14` (node 16.14.2)

---

## Steps to Reproduce

1. Create the following package.json:
```
{
  "name": "test",
  "version": "1.0.0",
  "devDependencies": {
    "npm-check-updates": "16.7.9",
    "outdated": "npm:npm-check-updates@12.3.1"
  }
}
```
2. Run `ncu --cache`.

## Current Behavior
NCU crashes with an error:
```
TypeError: Cannot read properties of null (reading '1')
    at .../node_modules/npm-check-updates/build/src/lib/upgradeDependencies.js:67:65
    at .../node_modules/npm-check-updates/node_modules/lodash/mapValues.js:38:34
    at .../node_modules/npm-check-updates/node_modules/lodash/_createBaseFor.js:17:11
    at baseForOwn (.../node_modules/npm-check-updates/node_modules/lodash/_baseForOwn.js:13:20)
    at mapValues (.../node_modules/npm-check-updates/node_modules/lodash/mapValues.js:37:3)
    at current.current (.../node_modules/npm-check-updates/build/src/lib/upgradeDependencies.js:60:43)
    at .../node_modules/npm-check-updates/node_modules/lodash/_createFlow.js:71:31
    at upgradeDependencies (.../node_modules/npm-check-updates/build/src/lib/upgradeDependencies.js:96:7)
    at upgradePackageDefinitions (.../node_modules/npm-check-updates/build/src/lib/upgradePackageDefinitions.js:29:68)
    at async runLocal (.../node_modules/npm-check-updates/build/src/lib/runLocal.js:166:65)
```

## Expected Behavior

NCU does not crash, suggests:
```
 npm-check-updates                        16.7.9  →  16.7.10
 outdated           npm:npm-check-updates@12.3.1  →  16.7.10
```

## Notes
Any package likely works for reproducing this, I'm using `npm-check-updates` itself here as an example. The package needs to be specified twice (normally and under an alias) and both versions need to be outdated.

The error occurs in this piece of code:
 https://github.com/raineorshine/npm-check-updates/blob/8ad8496404950d321a10e46ac101caa9b33b1511/src/lib/upgradeDependencies.ts#L61-L63

In the above scenario, `current` is an npm alias, but `latest` is not, so the second call to `parseNpmAlias` returns `null`

**Comment by raineorshine** [comment]
Thanks for the thorough report! It made it easy for me to make the fix.

Fixed and published in `v16.7.11`.

**Comment by P4** [comment]
Updated to `16.7.12`, running with `--cache` now works without issues. Thanks!
